client-js/src/scripts/generate-wasm.ts

```diff
@@ -27,12 +26,3 @@ const result = `
 `;
 
 fs.writeFileSync(path.resolve(__dirname, '../wasm.ts'), result);
```

Comment:
maybe we can [unify the scripts](https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/dev/client-js/scripts/copy-and-convert-wasms.ts) and even in future release a pkg or make this part of the sdk ? :)